packages/trip-form/src/MetroModeSelector/index.tsx

```diff
@@ -327,14 +365,20 @@ export default function ModeSelector({
   onSettingsUpdate,
   onToggleModeButton
 }: Props): ReactElement {
+  const [keyboardPopup, setKeyboardPopup] = useState(null);
```

Comment:
Can you specify the type of this? It seems to be storing a function or undefined.